src/redux/categories/categories.type.js

```diff
@@ -0,0 +1,7 @@
+const categoriesActionType = {
+  FETCH_CATEGORIES_START: 'FETCH_CATEGORIES_START',
+  FETCH_CATEGORIES_SUCCESS: 'FETCH_CATEGORIES_SUCCESS',
+  FETCH_CATEGORIES_FAILURE: 'FETCH_CATEGORIES_FAILURE',
+};
```

Comment:
- [ ] Kindly create `CHECK_STATUS` action type.

- [ ] The value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `CHECK_STATUS:  `'bookstore/categories/CHECK_STATUS'`